backend/celeryapp/Gemfile.lock

```diff
@@ -247,6 +247,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-22
+  arm64-darwin-24
```

Comment:
I wonder what this does!  && feel like it's just one more reason to dockerize ':D